cmd/cluster-operator/customcluster/customcluster.go

```diff
@@ -47,5 +49,11 @@ func InitControllers(ctx context.Context, opts *options.Options, mgr ctrl.Manage
 		return err
 	}
 
+	if err := (&webhooks.CustomClusterWebhook{
+		Client: mgr.GetClient(),
+	}).SetupWebhookWithManager(mgr); err != nil {
+		return fmt.Errorf("unable to create CustomCluster webhook, %w", err)
```

Comment:
keep consistent with other manager

Reply:
I'm sorry, I didn't fully understand your point. Could you please provide a more accurate description? For instance, are you referring to the error log?

Reply:
https://github.com/kurator-dev/kurator/pull/338/files#diff-20f968949a43d1be5cffd02f5c9b6f8aaf7e02cb9b95af36923c3a93f517fddeR39-R40